src/adagrams.js

```diff
@@ -1,8 +1,204 @@
+]
+
+// console.log(letterPool);
+
+const poolCreate = (letterPool) => {
+  const letterPoolToDrawFrom = [];
+  letterPool.forEach((letters) => { //for each array of letters of letterPool
+    letters.forEach((letter) => { // for each letter of array of letters
+      letterPoolToDrawFrom.push(letter)//push each letter into the pool of letter
+    }); 
+  });
+  return letterPoolToDrawFrom;
+};
+
+// console.log(poolCreate(letterPool));
+
+const shuffleLetterPool = (array) => {
+  // Starting at the end of the array/last index
+  // Move toward the first index/front of the array
+  for (let i = array.length - 1; i > 0; i--) {
+    // Math.random returns a random number between the range of 0...1 
+    // * by a number = (0*num)...(1*num)
+    // if array.length = 50, j is random num between 0 and 49
+    const j = Math.floor(Math.random() * (i + 1));
+    const temp = array[i];
+    array[i] = array[j];
+    array[j] = temp;
+  }
+  return array;
+};
+
 const Adagrams = {
-  drawLetters() {
+  drawLetters: function() {
     // Implement this method for wave 1
+    // const shuffleLetterPoolToDrawFrom = _.shuffle(poolCreate(letterPool));
+    const shuffleLetterPoolToDrawFrom = shuffleLetterPool((poolCreate(letterPool)));
+
+    const letterInHand = []
+    for (let i = 0; i < 10; i++) {
+      const drawLetter = shuffleLetterPoolToDrawFrom.pop();
+      letterInHand.push(drawLetter);
+    };
+    return letterInHand;
   },
+
+  usesAvailableLetters: function(wordCreated, letterInHand) {
+    // Wave 2
+    let letterInHandCopy = Array.from(letterInHand);
+    const word = wordCreated.toUpperCase().split('');
+
+    for (const char of word) {
+      if (!letterInHandCopy.includes(char)) {
+        return false;
+      } else {
+        letterInHandCopy.splice(letterInHandCopy.indexOf(char), 1);
+      }
+    }
+    return true;
+  },
+
+  scoreWord: function(wordCreated) {
+    // Wave 3
+    const word = wordCreated.toUpperCase().split('');
+    let value = 0;
+
+    for (const char of word) {
+      switch(char) {
+        case '':
+          value = 0;
+          break;
+        case 'A':
+        case 'E':
+        case 'I':
+        case 'O':
+        case 'U':
+        case 'L':
+        case 'N':
+        case 'R':
+        case 'S':
+        case 'T':
+          value += 1;
+          break;
+        case 'D':
+        case 'G':
+          value += 2;
+          break;
+        case 'B':
+        case 'C':
+        case 'M':
+        case 'P':
+          value += 3;
+          break;
+        case 'F':
+        case 'H':
+        case 'V':
+        case 'W':
+        case 'Y':
+          value += 4;
+          break;
+        case 'K':
+          value += 5;
+          break;
+        case 'J':
+        case 'X':
+          value += 8;
+          break;
+        case 'Q':
+        case 'Z':
+          value += 10;
+          break;
+        // default: 
+        //   alert (`Your word contains ${char} an invalid character. `);
```

Comment:
You probably meant `throw` here instead of `alert`:

```suggestion
        //   throw(`Your word contains ${char} an invalid character. `);
```

`throw` is the JS version of `raise`.